examples/ingress.ipd

```diff
@@ -41,7 +41,170 @@ def install(ctx):
         data=[corev1.Namespace(metadata=metav1.ObjectMeta(name=namespace))],
     )
 
-    tls = vault.read("secret/piedpiper.com")
+    tls = {
```

Comment:
This will allow us to test using the example code, which is great. However, the original purpose of this file was to illustrate how isopod is useful, and hardcoding values as such make the example less readable. I am thinking maybe we could extract this value to another file, say `example/fake-tls.star`, which will look like the following

```python
fake_tls = {
    "ca-bundle": "...",
    "crt": "...",
    "key": "...",
}
```

and in `examples/ingress.ipd`

```python
load("helpers.ipd", "env_from_secret", "container_port", ... )
load("fake-tls.ipd", "fake_tls")

...

# You may read TLS config from Vault, such as:
#
#     tls = vault.read("secret/piedpiper.com")
# 
# Here, we use a fake value to implement a runnable example without access to Vault.
tls = fake_tls

...
```